### ISSUE TITLE
About wpa krack issue

### ISSUE THREAD

**Issue by Tray-0** [maroviher/DPO_RT5572_LinuxSTA_2.6.1.3_20121022_mod#3]
Would u fix krack issue on RT5572 driver?
I really need this because I'm a supplier of RT5572, many customers complain about their wireless card not safe! They want update.
If you are considering, contact me by email trayx@qq.com

**Comment by Tray-0** [comment]
Here is the krack website
https://www.krackattacks.com/

**Comment by maroviher** [comment]
I do not have a time for it now. Why do not you use an official Linux driver drivers/net/wireless/ralink/rt2x00
?

**Comment by Tray-0** [comment]
Thanks for reply.
I have a little bit of confusing, What's official Linux driver? Or where to get it?
May be you don't know that, I have contacted the team of Ralink before, but they said they were not handle this anymore because team dismissed.
So I don't have much ways unless searching a gay well know driver to fix this issue.
If you are considering this, I think we can make a business.
Hope for reply.

**Comment by maroviher** [comment]
Yes, there is no Ralink anymore, but Mediatech. Last driver update was in 2012. There is an alternative Linux driver for old Ralink chipsets, that is supported from Liunx community. You can find the sorce code for it in Linux kernel source code under following path: drivers/net/wireless/ralink/rt2x00
In order to help you, I have to know a little more. I will write you a private email.
